### DISCUSSION TITLE
What are the dimensions of the sidepanel insets?

### DISCUSSION THREAD

**Discussion by gohma231** [aderusha/DDD-Printable-Wall-Control-System#73] [Q&A]
What are the dimensions of the sidepanel insets?

It looks like they are supposed to be 4mm x 10mm. Can someone confirm this? 

Here is the specific sidepanel I am printing: https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Sidepieces/Flats/3x0%20Flat%20Left.stl

**Comment by aderusha** [comment]
Here's the model: https://github.com/aderusha/DDD-Printable-Wall-Control-System/blob/main/Accessories/4x10x8mm%20Pin.stl

They're actually 3.8 x 9.8 to leave a bit of clearance.

**Reply by gohma231** [reply]
Looks like that specific pin is actually 3.9 x 9.9mm if Cura is to be believed (see image below).

Is it safe to assume that the sidepanels were designed by cutting the pin you linked out of them? If not, then what are the dimensions of the hollowed out parts of the sidepanel? I'm trying to verify that my printer is printing the STL file properly.

![image](https://github.com/aderusha/DDD-Printable-Wall-Control-System/assets/4106435/bd7ae6ae-8459-46de-81e6-1901ed74b66c)

**Comment by aderusha** [comment]
Ahh sorry my bad, there is in fact 0.1mm clearance, I was running from memory and maybe shouldn't do that!  Anyway, the cutouts are 4x10.  They were designed by creating a 4x10 pocket.

**Reply by gohma231** [reply]
No worries! Thank you for quick responses!
